src/main/resources/jupyter/init-actions.sh

```diff
@@ -247,6 +248,22 @@ if [[ "${ROLE}" == 'Master' ]]; then
       done
     fi
 
+    #Install lab extensions
+    if [ ! -z "${JUPYTER_LAB_EXTENSIONS}" ] ; then
+      for ext in ${JUPYTER_LAB_EXTENSIONS}
+      do
+        log 'Installing JupyterLab extension [$ext]...'
+        if [[ $ext == 'gs://'* ]]; then
+          gsutil cp $ext /etc
+          JUPYTER_EXTENSION_ARCHIVE=`basename $ext`
+          docker cp /etc/${JUPYTER_EXTENSION_ARCHIVE} ${JUPYTER_SERVER_NAME}:${JUPYTER_HOME}/${JUPYTER_EXTENSION_ARCHIVE}
+          retry 3 docker exec ${JUPYTER_SERVER_NAME} ${JUPYTER_SCRIPTS}/extension/jupyter_install_lab_extension.sh ${JUPYTER_HOME}/${JUPYTER_EXTENSION_ARCHIVE}
+        else
```

Comment:
I'm just thinking of this, but can we support `http://` or `https://` locations for lab extensions? We already do similar for nbextensions (see above). Terra likes this feature because they can host their own extensions, and not need to copy to a bucket.